.circleci/deployment/workflows.yml

```diff
@@ -1,133 +1,136 @@
 #workflows:
-  dev-deployment:
+  deployment:
     when:
-      << pipeline.parameters.run_dev_deployment >>
+      and:
+        - or:
+            - equal: [ master, << pipeline.git.branch >> ]
+            - equal: [ main, << pipeline.git.branch >> ]
+            - equal: [ develop, << pipeline.git.branch >> ]
+            # - equal: [ '2722/circleci', << pipeline.git.branch >> ] #DELETE BEFORE MERGE
```

Comment:
can these comments here and elsewhere be deleted @George-Hudson ?

Reply:
Thank you Alex. I meant to do this already.

Reply:
@ADPennington Is there anything else you have questions on for this?